README.md

```diff
@@ -1,5 +1,7 @@
 # PostmanGeneratorBundle
 
+**This bundle is not necessary anymore if you use API Platform 2.0 or superior.** API Platform now supports natively [Swagger](http://swagger.io/) and Postman is able to create collections from a Swagger documentation. 
```

Comment:
I would prefer to remove "_anymore_", and specify again the 2.0 version on the second sentence: `API Platform 2.0 now supports…`